### PR TITLE
Add rail-fence-cipher exercice

### PR DESCRIPTION
**Pull request by vmichalak**
It's my first contribution to Exercism: I ported the rail-fence-cipher exercice.
If there is a problem, please let me know

### CODE REVIEW

File: exercises/rail-fence-cipher/.meta/version.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+1.2.40
```
Comment (sjwarner-bp) [inline exercises/rail-fence-cipher/.meta/version.txt]: Is this correct? 40 seems awfully high :laughing: 

I think the file should also just be called `version` without the `.txt`

Comment (vmichalak) [reply exercises/rail-fence-cipher/.meta/version.txt]: Oh damn, it's not the Kotlin version here ? :3

**Comment by vmichalak** [comment]
No problem, I will continue to wear exercises and when the construction will work again, we will merge it :)

**Comment by vmichalak** [comment]
No problem, I will continue to port exercises and when the build will work again, we will merge it :)

**Pull request merged**